assets/js/modules/analytics/datastore/properties.test.js

```diff
@@ -281,5 +304,10 @@ describe( 'modules/analytics properties', () => {
 				expect( foundProperty ).toEqual( undefined );
 			} );
 		} );
+		describe( 'getPrimaryPropertyType', () => {
+			it( 'returns the primary property type from the store', () => {
+				expect( registry.select( STORE_NAME ).getPrimaryPropertyType( ) ).toEqual( 'ua' );
```

Comment:
@danielgent, please, use `.toBe( ... )` for primitive types instead of `.toEqual( ... )` which is intended for complex types like objects.

Reply:
The same applies to other occurrences of `.toEqual()` in your tests.